circle.yml

```diff
@@ -12,9 +12,10 @@ machine:
     testpilot.dev: 127.0.0.1
   services:
     - docker
-
   node:
     version: 6.2.0
+  environment:
+    TESTPILOT_STATIC_BRANCHES: development-static 1305-revamped-static
```

Comment:
It's the specific branch that this PR is on, deploying to my throwaway S3 bucket for a demo of this PR:

http://testpilot-static.lorchard.mozaws.net.s3-website-us-east-1.amazonaws.com/